src/_rules/static.js

```diff
@@ -187,3 +187,8 @@ export const touchAction = [
   ['touch-pinch-zoom', { 'touch-action': 'pinch-zoom' }],
   ['touch-manipulation', { 'touch-action': 'manipulation' }],
 ];
+
+export const safeArea = [
+  ['pb-safe', { 'padding-bottom': 'calc(32px + env(safe-area-inset-bottom, 0px))' }],
```

Comment:
This `32px` is a bit custom and we maybe can do some smartness there but I can't come up with anything quick here 🤔 Ideas? Or just leave it for now?

Reply:
I guess it's fine to leave it like this for now. 
But just from the top of my head - maybe in the future we could allow an arbitrary value that would be used to calculate the safe area inset, if provided? Then it would work like this:
- `pb-safe` -> `{ 'padding-bottom': 'env(safe-area-inset-bottom, 0px)' }`
- `pb-safe-[32]` -> `{ 'padding-bottom': 'calc(32px + env(safe-area-inset-bottom, 0px))' }`

Reply:
That is great suggestion. I will implement it directly 💪

Reply:
https://github.com/warp-ds/drive/pull/103/commits/9eb6e7ae9f86451f349be0473f2ea96f6c173191